dashboards-observability/public/components/event_analytics/explorer/save_panel/save_panel.tsx

```diff
@@ -149,11 +162,11 @@ export const SavePanel = ({
                   data-test-subj="eventExplorer__metricMeasureSaveComboBox"
                 />
               </EuiFormRow>
-              <EuiSpacer size="s" />
+              {/* <EuiSpacer size="s" />
               <EuiTitle size="xxs">
                 <h3>{'Labels'}</h3>
-              </EuiTitle>
-              <EuiFormRow>
+              </EuiTitle> */}
+              {/* <EuiFormRow>
```

Comment:
remove comments if unnecessary? unless we are still deciding on using labels

Reply:
let this component be commented for now(can be used later), need to finish some more work in events home page